common/table.js

```diff
@@ -218,7 +221,14 @@
                     return defer.resolve(true);
                 }).catch(function(err) {
                     if (vm.reference.uri !== uri) {
-                        return defer.resolve(false);
+                        defer.resolve(false);
+                    } else {
+                        vm.hasLoaded = true;
+                        vm.initialized = true;
+                        if(err.errorData && err.errorData.redirectPath && err.errorData.redirectPath != ''){
```

Comment:
What you are doing in this `if` condition should replace `...` that I typed in the above code block.